Cargo.toml

```diff
@@ -45,9 +45,9 @@ libp2p-tcp = { version = "0.13.0", path = "transports/tcp" }
 libp2p-websocket = { version = "0.13.0", path = "transports/websocket", optional = true }
 
 [dev-dependencies]
+anyhow = "1.0"
```

Comment:
Since this is an opinionated way to handle errors, I'd be in favour of not using it in examples.